genproto/pluginv2/datasource.pb.go

```diff
@@ -24,9 +24,9 @@ var _ = math.Inf
 const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
 
 type DatasourceRequest struct {
-	TimeRange            *TimeRange         `protobuf:"bytes,1,opt,name=timeRange,proto3" json:"timeRange,omitempty"`
-	Datasource           *DatasourceInfo    `protobuf:"bytes,2,opt,name=datasource,proto3" json:"datasource,omitempty"`
-	Queries              []*DatasourceQuery `protobuf:"bytes,3,rep,name=queries,proto3" json:"queries,omitempty"`
+	TimeRange            *TimeRange         `protobuf:"bytes,1,opt,name=timeRange" json:"timeRange,omitempty"`
```

Comment:
These changes looks suspicious. What version of protoc are you using? I was the one re-generating protobuf last time I think. Here's my version:

```bash
> protoc --version
libprotoc 3.9.1
```